### PR TITLE
new bidder & analytics adapter: Concert

### PR DESCRIPTION
**Pull request by walia**
## Type of change

- [x] New bidder and analytics adapter   

## Description of change

- test parameters for validating bids
```
  var adUnits = [
    {
      code: 'desktop_leaderboard_variable',
      mediaTypes: {
        banner: {
          sizes: [[1030, 590]]
        }
      }
      bids: [
        {
          bidder: "concert",
          params: {
            partnerId: 'test_partner'
          }
        }
      ]
    }
  ];
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- support@concert.io
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/2033

### CODE REVIEW

**Comment by harpere** [comment]
@walia  some of your tests are failing.  See the circle CI link above.

**Comment by walia** [comment]
@harpere Is there way to debug this test locally since its an IE 11 issue? I was not able to get this error when running the unit tests with `gulp test`.

**Comment by robertrmartinez** [comment]
@walia There is a lint error in your PR.

image


Please fix.

@afewcc can you please give this a review when you get a chance?

**Comment by bsheldon** [comment]
@harpere thanks for taking a look here, please advise on relevant IE test above from @walia when able.

**Comment by gglas** [comment]
@msm0504 reassigned to you - please have a look when you get a chance!

File: modules/concertBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      mediaTypes: {
+        banner: {
+          sizes: [[1030, 590]]
+        }
+      }
+      bids: [
+        {
+          bidder: "concert",
+          params: {
+            partnerId: 'test_partner'
```
Comment (msm0504) [inline modules/concertBidAdapter.md:27]: Can this be a `partnerId` that will return data? When I used these params in a test page, I got this response:
```
{"bids":[],"debug":{}}
```
Also, all responses from your endpoint should be compressed (such as gzip, compress, deflate)

**Pull request closed without merging**